metagraph/src/annotation/representation/column_compressed/annotate_column_compressed.hpp

```diff
@@ -166,7 +167,8 @@ class ColumnCompressed : public MultiLabelEncoded<Label> {
     uint8_t count_width_;
     uint64_t max_count_;
     std::vector<sdsl::int_vector<>> relation_counts_;
-    std::vector<Vector<std::pair<Index, uint64_t>>> coords_;
+    std::vector<common::SortedVector<std::pair<Index, uint64_t>>> coords_;
```

Comment:
maybe add a comment here to state that this may be stored either completely in RAM, or in a disk buffer depending on input parameters (at least, this is what I gathered from reading ahead)